### ISSUE TITLE
--webserver-port option doesn't change the actual port

### ISSUE THREAD

**Issue by idoshamun** [Polymer/web-component-tester#547]
I am using version 6.0.0 and when trying to change the webserver port using the `--webserver-port` argument is still remains 8081 as the default `polymer serve` port.

**Comment by hypnoce** [comment]
Indeed, the port is not passed during server creation (https://github.com/Polymer/web-component-tester/blob/master/runner/webserver.ts#L136)

Also, even if the hostname looks like it's passed, localhost is hard coded : https://github.com/Polymer/web-component-tester/blob/master/runner/webserver.ts#L184
The hostname issue has already been reported and fixed in another branch : https://github.com/Polymer/web-component-tester/issues/523

@Polymer team : is there any impediments to merge the use-hostname branch into master ?

**Comment by c-okelly** [comment]
The new pr fixes the issues and ensure ensure that the correct default value is used when the user does not set the port.
